plugin/src/main/kotlin/tanvd/grazi/language/Lang.kt

```diff
@@ -61,8 +82,10 @@ enum class Lang(val displayName: String, val shortCode: String, private val clas
             tool.enableRule(it.id)
         }
 
+        // disable all spellchecker rules also (use them directly in GraziSpellchecker)
         tool.allRules.filter { rule ->
-            disabledRules.any { rule.id.contains(it) } || disabledCategories.any { rule.category.id.toString().contains(it) }
+            rule.isDictionaryBasedSpellingRule || disabledRules.any { rule.id.contains(it) }
```

Comment:
Why not to disable rule during creation of `LangTool`?

Reply:
Well, configure is called first during creation of LangTool :) Why we need to iterate all rules twice?